src/components/Table/GridTable.tsx

```diff
@@ -61,8 +61,19 @@ export interface GridStyle {
   firstRowMessageCss?: Properties;
   /** Applied on hover if a row has a rowLink/onClick set. */
   rowHoverColor?: string;
-  /** Styling for nested cards (see `cardStyle` if you only need a flat list of cards). */
-  nestedCards?: Record<string, NestedCardStyle>;
```

Comment:
Made a new `NestedCardsStyle` to have both the `topLevelSpacerPx` + the previous/existing per-kind map to `NestedCardStyle`.